### ISSUE TITLE
Corrupting sudo with system commands part of the instruction

### ISSUE THREAD

**Issue by ozyreusz** [foosel/OctoPrint#1711]
You don't need this part

```
 If you want to be able to shutdown and restart your Pi via the webinterface, you'll first have to add a sudo` rule for the system user OctoPrint is running under (for me that's the default user pi):

pi@raspberrypi ~ $ sudo -s
root@raspberrypi:/home/pi# cat > /etc/sudoers.d/octoprint-shutdown
pi ALL=NOPASSWD: /sbin/shutdown

Then hit Ctrl+D twice to first exit the cat command and then the root shell
```
There already is access in /etc/sudoers.d/ allowing all applications to use sudo. Making a second file coses to create a corrupt sudo which has to many sudoers. It's very hard to fix remotely. Default state allows to make sudo by everyone logged in

**Comment by GitIssueBot** [comment]
Hi @ozyreusz,

It looks like there is some **information missing** from your bug report that will be needed in order to solve the problem. Read the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md) which will provide you with a template to fill out here so that your bug report is ready to be investigated (I promise I'll go away then too!).

If you did not intend to report a bug but wanted to **request a feature or brain storm** about some kind of development, please take special note of the title format to use as described in the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md).

**Please do not abuse the bug tracker as a support forum** - if you have a question or otherwise need some kind of help or support refer to the [Mailinglist](https://groups.google.com/group/octoprint) or the [G+ Community](https://plus.google.com/communities/102771308349328485741) instead of here.

Also **make sure you are at the right place** - this is the bug tracker of the official version of OctoPrint, not the Raspberry Pi image OctoPi nor any unbundled third party OctoPrint plugins or unofficial versions. Make sure too that you have **read through the [Frequently Asked Questions](https://github.com/foosel/OctoPrint/wiki/FAQ)** and searched the [**existing tickets**](https://github.com/foosel/OctoPrint/search?q=&ref=cmdform&type=Issues) for your problem - try multiple search terms please.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two weeks (until 2017-01-28 08:00 UTC) I'll close this ticket so it doesn't clutter the bug tracker. This is nothing personal, so please just be considerate and help the maintainers solve this problem quickly by following the guidelines linked above. Remember, the less time the devs have to spend running after information on tickets, the more time they have to actually solve problems and add awesome new features. Thank you!

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being, so don't expect any replies from me :) Your ticket is read by humans too, I'm just not one of them.

**Comment by foosel** [comment]
> Making a second file coses to create a corrupt sudo which has to many sudoers.

No, adding the additional sudoers file as described in the wiki page doesn't cause any corruption at all, unless the file you add is broken itself. If you do create a valid file (as in, follow the guide to the T), no corruption happens. That file ensures that the two commands in question will be executable by the `pi` user even if the user happens to have disabled the (somewhat idiotic) default configuration in Raspbian that allows passwordless sudo for all commands. If it corrupted something for you, it was due to a copy-paste error in the file or some other kind of error. OctoPi even ships with those exact files and that works fine for thousands of users out there (myself included).

I edited the _wiki_ though to clarify that that step isn't needed unless one has modified the standard sudo configuration to be more secure than what ships with Raspbian by default.
